plugins/api-docs/package.json

```
@@ -67,7 +67,7 @@
     "@material-ui/core": "^4.12.2",
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "4.0.0-alpha.61",
-    "graphiql": "3.1.1",
+    "graphiql": "3.7.1",
```

Why is this version locked in the first place? Let's unlock it.

```suggestion
    "graphiql": "^3.7.1",
```